non-atomic-action-interpreter-module/.gitignore

```diff
@@ -0,0 +1,10 @@
+.idea
+.vscode
+cmake-build-*
+/build
+/bin
+/ConanPresets.json
+
```

Comment:
can you add them to gitignore?
![image](https://github.com/user-attachments/assets/f542600f-5291-438d-bb3a-14b5b09a0d35)

Reply:
can you place them in this module? when it'll be installed as a folder folder should contain those files in gitignore